### PR TITLE
🌱  Add debug logging around CombinedStatus resolution

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR makes some debug logging improvements in the status controller.

## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/status/controller.go [hunk 1]
```diff
@@ -199,18 +197,20 @@ func (c *Controller) run(ctx context.Context, workers int, cListers chan interfa
 
 	c.bindingResolutionBroker.RegisterCallback(func(bindingPolicyKey string) {
 		// add binding to workqueue
+		logger.V(5).Info("Enqueuing reference to Binding due to notification from BindingResolutionBroken", "name", bindingPolicyKey)
```
Comment (vMaroon) [inline pkg/status/controller.go:200]: s/Broken/Broker/.

Comment (MikeSpreitzer) [reply pkg/status/controller.go]: Fixed, thanks!

**Pull request merged**